source/codegen/templates/service_helpers.mako

```diff
@@ -638,9 +638,10 @@ ${initialize_standard_input_param(function_name, parameter)}
   parameter_name = common_helpers.get_cpp_local_name(parameter)
   underlying_param_type = common_helpers.get_underlying_type_name(parameter["type"])
   grpc_type = parameter["grpc_type"]
+  grpc_cpp_lib_type = service_helpers.get_protobuf_cpplib_type(grpc_type)
```

Comment:
Note: Changes in this file were taken directly from https://github.com/ni/grpc-device/pull/607, will go away after that goes in and this branch is rebased.